pkg/apis/operator/v1alpha1/additional_options.go

```diff
@@ -25,9 +25,10 @@ import (
 // additional options will be updated on the manifests
 // these values will be final
 type AdditionalOptions struct {
-	Disabled                 *bool                                            `json:"disabled,omitempty"`
-	ConfigMaps               map[string]corev1.ConfigMap                      `json:"configMaps,omitempty"`
-	Deployments              map[string]appsv1.Deployment                     `json:"deployments,omitempty"`
-	HorizontalPodAutoscalers map[string]autoscalingv2.HorizontalPodAutoscaler `json:"horizontalPodAutoscalers,omitempty"`
-	StatefulSets             map[string]appsv1.StatefulSet                    `json:"statefulSets,omitempty"`
+	Disabled                    *bool                                            `json:"disabled,omitempty"`
+	ConfigMaps                  map[string]corev1.ConfigMap                      `json:"configMaps,omitempty"`
+	Deployments                 map[string]appsv1.Deployment                     `json:"deployments,omitempty"`
+	HorizontalPodAutoscalers    map[string]autoscalingv2.HorizontalPodAutoscaler `json:"horizontalPodAutoscalers,omitempty"`
+	StatefulSets                map[string]appsv1.StatefulSet                    `json:"statefulSets,omitempty"`
+	WebhookConfigurationOptions map[string]*WebhookConfigurationOptions          `json:"webhookConfigurationOptions,omitempty"`
```

Comment:
@jkhelil as you have modified on api, you need to run `make generated` and include deepcopy changes

Reply:
intersting fact, update-codegen is not generating anything on macos, had to start a linux vm to run it

Reply:
@jkhelil Seems we will not get a change on this, I see the `build tests` are passing